pipelines/nextflow/modules/fasta/process_fasta_data.nf

```diff
@@ -27,10 +27,11 @@ process PROCESS_FASTA {
 
     shell:
     output_fasta = pep_mode == 1 ? "fasta_pep.fa" : "fasta_dna.fa"
+    pep_mode_flag = pep_mode == 0 ? "" : "--peptide_mode"
     '''
     prep_fasta_data --fasta_infile !{fasta_file} \
```

Comment:
a bit confusing, should we change this (and in the related modules) to `process_fasta`?  and update pyproject's script entry point as well?

Reply:
I will make sure we cover this in tomorrow's meeting :+1: